cognite/neat/graph/stores/_base.py

```diff
@@ -361,3 +362,18 @@ def list_instances_of_type(self, class_uri: URIRef) -> list[ResultRow]:
         logging.info(query)
         # Select queries gives an iterable of result rows
         return cast(list[ResultRow], list(self.store.query(query)))
+
+    def triples_of_type(self, rdf_type: str) -> list[tuple[str, str, str]]:
+        """Get all triples of a given type.
```

Comment:
with this method you are getting all triples defining all instances of a given type, so maybe the method name cloud be `triples_of_type_instances` ?